addons/stock/report/report_deliveryslip.xml

```diff
@@ -81,7 +81,7 @@
                                     <span t-field="move.product_uom_qty"/>
                                     <span t-field="move.product_uom"/>
                                 </td>
-                                <td>
+                                <td name="move_quantity_done">
```

Comment:
I through you added this name to use it in the xpath but I don't see it anywhere. Why this change ?